### PR TITLE
10 May homepage content programming update

### LINKED ISSUE #4903: 10 May homepage content programming update
For the **10 May** push, please find homepage content updates detailed [here](https://docs.google.com/document/d/1g4RuGKW__eTHNd277rKPMM46ygonXzW68jSs2OXEkPM/edit#). 

@muffinresearch @diox @bobsilverberg

### PR DESCRIPTION
**Pull request by bobsilverberg**
Fixes #4903

### CODE REVIEW

File: src/amo/components/HomeHeroBanner/index.js [hunk 1]
```diff
@@ -233,7 +228,7 @@ export class HomeHeroBannerBase extends React.Component<Props> {
       ),
       (
         <HeroSection
-          key="hero-19"
+          key="hero-18"
           linkTo="/addon/privacy-badger17/"
         >
```
[the remark this answers is not in the document]
Comment (bobsilverberg) [reply src/amo/components/HomeHeroBanner/index.js]: We do, but it doesn't test the content of all of the possible HeroSections. When I first started doing this I just went with the flow. I suppose we could add a test that verifies all of the HeroSections, but that seems like a make work project. All we'll really be doing is writing the same code twice, once in the component and once in the test, and if would be super easy for someone to copy and paste and introduce the same error in both files, in which case the test would be invalid.

I'm in favour of keeping it as is and not adding a test for the content of each HeroSection. Do you agree?

File: src/amo/components/HomeHeroBanner/index.js [hunk 2]
```diff
@@ -116,24 +119,26 @@ export class HomeHeroBannerBase extends React.Component<Props> {
...
-          linkTo="/addon/tab_search/"
+          linkTo="/addon/envify/"
         >
-          <h3>{i18n.gettext('TabSearch')}</h3>
+          <h3>{i18n.gettext('Envify')}</h3>
 
           <p>
-            {i18n.gettext('Need a search function for just your open tabs?')}
+            {i18n.gettext(`Quantum Extensions Challenge winner!
+              Different tab colors for different dev environment`)}
```
[the remark this answers is not in the document]
Comment (bobsilverberg) [reply src/amo/components/HomeHeroBanner/index.js]: Good catch, thanks!

File: src/amo/components/HomeHeroBanner/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (willdurand) [reply src/amo/components/HomeHeroBanner/index.js]: I do.

[the remark this answers is not in the document]
Comment (willdurand) [reply src/amo/components/HomeHeroBanner/index.js]: Do we have a smoke test to ensure that we actually have `HeroSection` though?

[the remark this answers is not in the document]
Comment (bobsilverberg) [reply src/amo/components/HomeHeroBanner/index.js]: We have https://github.com/mozilla/addons-frontend/blob/master/tests/unit/amo/components/TestHomeHeroBanner.js#L28-L34. Does that seem sufficient?

[the remark this answers is not in the document]
Comment (bobsilverberg) [reply src/amo/components/HomeHeroBanner/index.js]: Also, I always check it manually, as does QA, but I guess we also want full automated test coverage if possible.

[the remark this answers is not in the document]
Comment (willdurand) [reply src/amo/components/HomeHeroBanner/index.js]: ok, perfect 👍

**Pull request merged**